docs/sql-ref-datatypes.md

```diff
@@ -119,10 +119,10 @@ from pyspark.sql.types import *
 
 |Data type|Value type in Python|API to access or create a data type|
```

Comment:
Actually on second thought, can we just add a link back from Python docs to here? then we won't need to duplicate the table.